### PR TITLE
Add AutoYaST self_update element

### PR DESCRIPTION
**Pull request by imobachgs**
This PR replaces #102. It adds the description of a new AutoYaST element `self_update`. See yast/yast-installation#501 for more details.

### CODE REVIEW

File: xml/ay_bigfile.xml [hunk 1]
```diff
@@ -1029,6 +1029,37 @@ exit 0
       <row>
        <entry>
         <para>
+         <literal>self_update</literal>
+        </para>
+       </entry>
+       <entry>
+        <para>
+         This option allows to enable/disable the self-update.
```
Comment (fsundermeyer) [inline xml/ay_bigfile.xml]: Better:

    This option allows to enable (set it to <literal>true</literal>)
    or disable (set it to <literal>false</literal> the self-update.

File: xml/ay_bigfile.xml [hunk 2]
```diff
@@ -1029,6 +1029,37 @@ exit 0
...
+         Check out the &deploy; to find further information about this feature.
```
Comment (fsundermeyer) [inline xml/ay_bigfile.xml]: Please replace `&deploy;` with the following:

    <xref linkend="sec.i.yast2.self_update"/>

The will create a link to the respective section in the deployment guide.

File: xml/ay_bigfile.xml [hunk 3]
```diff
@@ -1029,6 +1029,37 @@ exit 0
...
+         <literal>self_update</literal> on the Kernel command line.
+        </para>
+        <important>
+         <title>Disabling the self-update</title>
+         <para>
+          Please, note that this feature will be skipped if any option disables
+          it: setting this option to <literal>false</literal> or using the
+          boot parameter <literal>self_update=0</literal>.
+         </para>
+        </important>
```
Comment (fsundermeyer) [inline xml/ay_bigfile.xml]: If you document true/false and 0/1 as suggested above, the complete <important> section can be removed.

**Comment by imobachgs** [comment]
I'm closing this PR (and opening a new one as #105) because I made an stupid mistake and deleted my old fork without noticing that a PR was still open. Sorry for the noise.

**Pull request closed without merging**